sklearn/linear_model/tests/test_logistic.py

```diff
@@ -1419,15 +1419,17 @@ def test_n_iter(solver):
     assert clf.n_iter_.shape == (1, n_cv_fold, n_Cs)
 
     # multinomial case
-    n_classes = 1
```

Comment:
I think `n_classes` was used here to be consistent with the code above involving the `'ovr'` case. Above there is a conditional on solver to set `n_classes`:

https://github.com/scikit-learn/scikit-learn/blob/845b1fac9e8bcfc146e7b38119ce6c500e00a10b/sklearn/linear_model/tests/test_logistic.py#L1400

Overall, I am happy with this PR.

Reply:
Indeed I need to change the beginning of the test as well...

Reply:
I discovered that there was no good reason for liblinear to behave differently from the others in the OvR case. I fixed it and will add a changelog entry.